src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java

```diff
@@ -2,20 +2,16 @@
 
 import seedu.lifeasier.storage.FileStorage;
 import seedu.lifeasier.tasks.TaskList;
-import seedu.lifeasier.tasks.Task;
-import seedu.lifeasier.tasks.Deadline;
-import seedu.lifeasier.tasks.Event;
-import seedu.lifeasier.tasks.Lesson;
 import seedu.lifeasier.ui.Ui;
```

Comment:
Should this import be removed if we are no longer using the Ui class?

Reply:
But isn't Ui being passed into the execute method of all Command classes?

Reply:
Oh yes I forgot about that. My bad!